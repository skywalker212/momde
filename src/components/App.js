import React, { Component } from 'react';

class App extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <h1 className="App-title">My Own MarkDown Editor</h1>
        </header>
        <p className="App-intro">
          let's write some MarkDown MOFOS!
        </p>
      </div>
    );
  }
}

export default App;
