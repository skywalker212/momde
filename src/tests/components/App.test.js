import React from 'react';
import App from '../../components/App';
import {shallow} from 'enzyme';

it('renders without crashing', () => {
  const wrapper = shallow(<App/>);
  expect(wrapper).toMatchSnapshot();
});
