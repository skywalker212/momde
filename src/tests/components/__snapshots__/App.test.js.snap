// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h1
      className="App-title"
    >
      My Own MarkDown Editor
    </h1>
  </header>
  <p
    className="App-intro"
  >
    let's write some MarkDown MOFOS!
  </p>
</div>
`;
